### ISSUE TITLE
BART : Denoising Sequence-to-Sequence Pre-training for Natural Language Generation , Translation , and Comprehension

### ISSUE THREAD

**Issue by gori-san** [gori-san/paper100knock#5]
## 概要
Transformer-based のSeq2Seq の事前学習に関する論文。B[E]RT ではなく B[A]RT である。Token Masking、Token Deletion、Text Infilling (スパンのマスク)、Sentence Permutation (文の順番の shuffle)、Document Rotation (ランダムに選択されたトークンから文書を始める) からの文の復元を事前学習している。
fine-tuning の際に、classification はモデルをそのまま Encoder-Decoder として用いるが、翻訳では事前学習したモデルをターゲット側の言語の Encoder-Decoder として用い、その前にランダムに初期化したソース側の言語の Encoder を用意して全体を fine-tuning している。
要約 (XSum、CNN/DN) や対話生成 (ConvAI2)、生成型 QA (ELI5) などの生成系のタスクにおいて SOTA となっている。

### リンク
https://www.aclweb.org/anthology/2020.acl-main.703/

### 著者
Mike Lewis, Yinhan Liu, Naman Goyal, Marjan Ghazvininejad, Abdelrahman Mohamed, Omer Levy, Veselin Stoyanov, Luke Zettlemoyer
[Facebook AI]

### 実装
https://github.com/pytorch/fairseq/tree/master/examples/bart
https://huggingface.co/transformers/model_doc/bart.html#

### その他
ACL 2020 long paper
